zio-kafka-test-utils/src/main/resources/kafka-generate-ssl.sh

```diff
@@ -0,0 +1,207 @@
+#!/usr/bin/env bash
```

Comment:
@gurinderu Did you implement this or did you copy it from somewhere?

Reply:
copied from https://github.com/confluentinc/confluent-platform-security-tools/blob/master/kafka-generate-ssl.sh 
but yep, probably need to drop it here

Reply:
Please add a comment telling where this scripts comes from :)

Reply:
Removed

Reply:
I'd prefer we keep this script and we just document its origin

Reply:
Not sure that is a great idea in case of licenses